lib/trackler/track.rb

```diff
@@ -102,6 +103,10 @@ def slugs
       active_slugs + foregone_slugs + deprecated_slugs
     end
 
+    def unimplemented_problems
```

Comment:
Exercism should do this logic, I think. I'm not convinced it belongs in `Trackler::Track`.